linux/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "headset",
-  "productName": "Headset",
+  "productName": "headset",
```

Comment:
you should leave this as **Headset**, you're allowed to use uppercase here. Otherwise, your app will be known as headset everywhere and not just the executable.